Makefile

```diff
@@ -206,7 +206,11 @@ $(VENDOR_DIR): glide.lock glide.yaml
 deps: $(VENDOR_DIR)
 
 app/controllers.go: $(DESIGNS) $(GOAGEN_BIN) $(VENDOR_DIR)
-	$(GOAGEN_BIN) bootstrap -d ${PACKAGE_NAME}/${DESIGN_DIR}
+	$(GOAGEN_BIN) app -d ${PACKAGE_NAME}/${DESIGN_DIR}
+	$(GOAGEN_BIN) controller -d ${PACKAGE_NAME}/${DESIGN_DIR} -o controllers/ --pkg controllers --app-pkg app
```

Comment:
well, you can look at the `models` pkg in our repo. But you could easily find other repos with plural names.